src/utils/createObjectMethodTransform.js

```diff
@@ -41,6 +41,20 @@ export default function createObjectMethodTransform(polyfill) {
         return
       }
 
+      if (
+        t.isMemberExpression(path.node.callee) &&
+        t.isIdentifier(path.node.callee.property) &&
+        t.isMemberExpression(path.node.arguments[0]) &&
+        path.node.arguments[0].object.name === object &&
+        path.node.arguments[0].property.name === property
+      ) {
+        path.replaceWith(
+          t.callExpression(path.node.callee, [
+            createImportIdentifier(file, polyfill),
+          ])
```

Comment:
There is a problem with this. You are replacing all the arguments with the imported identifier instead of only the one that is required.

```js
[1, 2, 3].reduce(Math.sign, 0)

// Will be replaced with

[1, 2, 3].reduce(_Math_sign)

// Instead of
[1, 2, 3].reduce(_Math_sign, 0)
```